dev/breeze/src/airflow_breeze/commands/developer_commands_config.py

```diff
@@ -362,4 +361,12 @@
             ],
         },
     ],
+    "breeze doctor": [
+        {
+            "name": "Debug breeze",
```

Comment:
```suggestion
            "name": "Auto-healing of breeze",
```

Reply:
doctor heals, not debugs :)

Reply:
Now, what can we use "breeze inject" for

Reply:
Updated this!